data.yml

```diff
@@ -1707,21 +1686,29 @@ components:
       required:
       - type
       - id
+      - creationDate
+      - lastUpdated
+      - status
+      - name
+      - bsb
+      - unmaskedAccNum
+      - maskedNumber
       - accountNo
+      - accountHolder
+      - isOwned
       - accountOwnership
-      - name
-      - currency
+      - bundleName
       - balance
       - availableFunds
-      - lastUpdated
-      - institution
       - creditLimit
+      - amortisedLimit
```

Comment:
Maybe next sprint, have something that specifies what this is.

Reply:
@basiq-chris its defined in the description and also taken from [here](https://basiq.atlassian.net/wiki/spaces/COM/pages/1892614434/Open+Banking+Account+Data~text=Object%20representing%20the%20available%20limit%20amortised%20according%20to%20payment%20schedule.%20Assumed%20to%20be%20zero%20if%20absent).

Reply:
defined [here](https://github.com/basiqio/api-ref/pull/331/files#diff-093f056f1d238f3fb59742e58ef57a2a0d056e07e42e18a72b451da2801a489eR1763). @basiq-chris

Reply:
Meant more what the word `amortised` meant

Reply:
sure, will add in the next sprint.